### PR TITLE
Not very correct name printing

### PR DESCRIPTION
**Pull request by favonia**

### CODE REVIEW

File: src/core/Name.ml [hunk 1]
```diff
@@ -1,51 +1,29 @@
 let counter = ref 0
 let names = Hashtbl.create 1000
 
-type t =
-  | Gen of int
-  | Const of string
-
-let const str =
-  Const str
+type t = int
```
Comment (favonia) [inline src/core/Name.ml]: @jonsterling LOL

Comment (jonsterling) [reply src/core/Name.ml]: lmao!!!

**Review by jonsterling — APPROVED** [review]
Looks like a big improvement! :laughing:

**Comment by favonia** [comment]
Partially address RedPRL/algaett#25.

**Pull request merged**